axonserver/src/main/java/io/axoniq/axonserver/message/event/EventDispatcher.java

```diff
@@ -295,13 +291,14 @@ public void onCompleted() {
 
     public void getFirstToken(GetFirstTokenRequest request, StreamObserver<TrackingToken> responseObserver0) {
         ForwardingStreamObserver<TrackingToken> responseObserver = new ForwardingStreamObserver<>(logger, "getFirstToken", responseObserver0);
-        checkConnection(contextProvider.getContext(), responseObserver).ifPresent(client ->
+        checkConnection(contextProvider.getContext(), responseObserver, false).ifPresent(client ->
             client.getFirstToken(contextProvider.getContext(), request, responseObserver)
         );
     }
 
-    private Optional<EventStore> checkConnection(String context, StreamObserver<?> responseObserver) {
-        EventStore eventStore = eventStoreLocator.getEventStore(context);
+    private Optional<EventStore> checkConnection(String context, StreamObserver<?> responseObserver,
+                                                 boolean useLocalStore) {
```

Comment:
It is always `false`, right? Maybe we can evaluate if the parameter is really needed.

Reply:
This is a leftover. The first implementation involved much more gRPC methods. Will fix it!